libbeat/scripts/kibana_import.py

```diff
@@ -1,47 +0,0 @@
-from elasticsearch import Elasticsearch
```

Comment:
I moved (and adjusted) the bash and powershell scripts in the beat repository, so no need to have another python script to do the same thing.